src/code.lisp

```diff
@@ -65,9 +116,15 @@
                              &key
                                (columns '())
                                (header (apply #'vellum.header:make-header
-                                              columns)))
+                                              columns))
+                               (parameters '()))
   (apply #'vellum:to-table
-         (postgres-query input header)
+         (etypecase input
```

Comment:
This is a weird code to write. Etypecase is supposed to emit an error when no case matches, but the last type is T. Perhaps replace it with  postgres-simple-query?

Reply:
Done.